eval/bundled/epm.elv.go

```diff
@@ -55,7 +55,7 @@ fn dest [pkg]{
 }
 
 fn is-installed [pkg]{
-  put ?(test -e (dest $pkg))
+  put (or (and ?(test -e (dest $pkg)) $true) $false)
```

Comment:
There is a `bool` [builtin](https://elvish.io/ref/builtin.html#bool) for this :)

Reply:
Duh - of course, thanks for the reminder! :)